services/sidecar/Dockerfile

```diff
@@ -42,15 +50,16 @@ VOLUME /home/scu/log
 
 FROM base as build
 
-ENV SC_BUILD_TARGET build
+ENV SC_BUILD_TARGET=build
 
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
+      build-essential \
+      gcc
 
-RUN apk add --no-cache \
-      postgresql-dev \
-      gcc \
-      libc-dev
+RUN python -m venv ${VIRTUAL_ENV}
 
-RUN $SC_PIP install --upgrade \
+RUN pip install --upgrade --no-cache-dir \
```

Comment:
MINOR: ``--no-cache-dir`` is a global option, I would write: `` pip --no-cache-dir install ...``

Reply:
done